TESTING.md

```diff
@@ -79,6 +79,49 @@ The detected frameworks would be:
 
 If no `frameworks` folder exists, the current folder is assumed to be the root of a single framework. Note that the name of the framework displayed in this case (and above) is purely cosmetic, and the package name for the framework is determined from the catalog definition files.
 
+#### Running tests interactively
+
+The default when invoking `test.sh` is to run all tests defined for a given framework. It may, however be desirable to run a subset of tests interactively while developing a feature or the related tests themselves.
+
+The following command:
+```bash
+CLUSTER_URL=$(dcos config show core.dcos_url) ./test.sh -i
```

Comment:
I wonder whether it would make sense that when `-i` is specified, then if CLUSTER_URL is unset then the script should default to using current dcos-attached cluster rather than create one automatically?

Reply:
d95ac60

Reply:
Doesn't it? Is https://github.com/mesosphere/dcos-commons/blob/6538663/test.sh#L99-L100 stale?

Reply:
Can you please drop the `CLUSTER_URL=$(dcos config show core.dcos_url) ` part from this line now that the script does it automagically?